eq-author/src/components/Header/index.test.js

```diff
@@ -141,7 +142,13 @@ describe("withCurrentUser", () => {
     };
 
     expect(
-      shallow(<WrappedComponent {...defaultProps} match={match} />)
+      shallow(
+        <WrappedComponent
+          {...defaultProps}
+          match={match}
+          title={"test-title"}
```

Comment:
This can be just a string: `title="test-title"`. Surprised the linter doesn't pick up this.

Reply:
Done.